src/plugins/vis_type_vislib/public/vislib/lib/handler.js

```diff
@@ -26,6 +27,10 @@ const markdownIt = new MarkdownIt({
   linkify: true,
 });
 
+const convertToTimestamp = (dateString) => {
+  return parseInt(moment(dateString).format('x'));
```

Comment:
This handles both when `dateString` is an ISO string and when it is just a number? (in case we have a number range on the date axis)?

Reply:
Yes Anton this is the reason that I preferred moment here, but I will change the `dateString` to date to be more clear.